templates/website/base.html

```diff
@@ -156,6 +156,7 @@
                 <div class="navmenu">
                     <ul class="menulist">
                         <li><a href="{% url 'website:home' %}" class="nav-link active">Home</a></li>
+                        <li><a href="{% url 'website:search_by_booking_code' %}" class="nav-link active">Search By Booking Code</a></li>
```

Comment:
This should only be accessed by logged in users so I would move this to line https://github.com/amakarudze/bnb/blob/main/templates/website/base.html#L108 and also this line https://github.com/amakarudze/bnb/blob/main/templates/website/base.html#L194. I would rename the link to `Search Reservation` and then tell the user to enter the booking code in the template. Otherwise, good work.